Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -864,6 +1024,36 @@ protected void onPostExecute(ObjectCursor<Note> cursor) {
         }
     }
 
+    private class refreshListForSearchTask extends AsyncTask<Void, Void, ObjectCursor<Note>> {
```

Comment:
shouldn't this be static to avoid memory leaks?

Reply:
Yeah, that's what lint suggests.  The problem with making it `static` is that all methods and variables within that class must be `static` as well.  So `setNoteSelected()`, `queryNotesForSearch()`, `getActivity()`, `requireActivity()`, `mNotesAdapter`, and `mSelectedNoteId` would need to be changed.  Since all the inner classes that extend `AsyncTask` in the codebase have this same lint warning, we should address all of them is a technical debt cleanup project.  I created https://github.com/Automattic/simplenote-android/issues/864 to address those.